test/TritonGPU/combine.mlir

```diff
@@ -1202,3 +1202,92 @@ module attributes {"triton_gpu.num-warps" = 4 : i32} {
     tt.return
   }
 }
+
+// -----
+
+// Check if MoveConvertOutOfLoop hangs because of adding additional conversions
+// CHECK-NOT: triton_gpu.convert_layout
```

Comment:
minor detail: In general you want a `CHECK` after `CHECK-NOT` otherwise it can interfere with the next test in the file. You could add a `//  CHECK: tt.return`

Reply:
Thanks, I agree.

I forgot to add `// CHECK: <function name>`...